src/index.ts

```diff
@@ -61,26 +93,37 @@ export function feedItemToMarkdownPost(item: Parser.Item & ExtraFields): Markdow
     author,
     content,
     contentSnippet: contentSnippet || "",
+    extraFieldValues,
   }
 }
 
-async function writeMarkdownPost(outDir: string, { title, author, originalLink, contentSnippet, content, pubDate }: MarkdownPost) {
+async function writeMarkdownPost(outDir: string, { title, author: origAuthor, originalLink, contentSnippet, content, pubDate, extraFieldValues }: MarkdownPost) {
   // Hack so that the content doesn't terminate the front matter
   content = content.replace(/[+]{3}/g, "\\u002B\\u002B\\u002B")
 
+  // Allow the extraFieldValues to overide the author, which allows us to deal
+  // with those blogs which don't send an author field.
+  const author = extraFieldValues.author ? extraFieldValues.author : origAuthor
```

Comment:
Is the only purpose of extraFieldValues for author? Would it be better if author was a separate flag so we don’t lose the type information?

Reply:
I'd also like to add "Blog Name" as an extra field, like "Mediocre Blog" or "Partially Functional" or whatever. I figure it'd be better to display the blog name rather than (or in addition to) author for our use-case, since a single blog might have multiple authors.